script/setup/install-protobuf

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Downloads and installs protobuf
+#
+set -eux -o pipefail
```

Comment:
Remove `-x` by default? could be enabled with `$DEBUG`, but I'm not sure if it's necessary.